api/v1/helper.go

```diff
@@ -271,6 +279,20 @@ func NeedToUpdateSriov(ifaceSpec *Interface, ifaceStatus *InterfaceExt) bool {
 							return true
 						}
 
+						if strings.EqualFold(ifaceStatus.LinkType, consts.LinkTypeETH) {
+							if len(vfStatus.Mac) == 0 {
+								log.V(2).Info("NeedToUpdateSriov(): VF MAC Address needs update",
+									"vf", vfStatus.VfID, "current", vfStatus.Mac)
+								return true
```

Comment:
we also need to reconcile if node/port guid is zeroes if device is rdma capable.

you may have VF admin mac set BUT node/port guid is unset (since driver was not unbind/bind)

root@cloud-dev-45:~# cat /sys/bus/pci/devices/0000\:03\:00.2/infiniband/mlx5_4/node_guid
0000:0000:0000:0000
root@cloud-dev-45:~# echo 0000:03:00.2 > /sys/bus/pci/drivers/mlx5_core/unbind
root@cloud-dev-45:~# echo 0000:03:00.2 > /sys/bus/pci/drivers/mlx5_core/bind
root@cloud-dev-45:~# cat /sys/bus/pci/devices/0000\:03\:00.2/infiniband/mlx5_4/node_guid
0012:34ff:fe56:7891